### ISSUE TITLE
Incorrect grpc-gateway dependency

### ISSUE THREAD

**Issue by ping-localhost** [sourcegraph/prototools#20]
`protoc-gen-doc` currently relies on `github.com/gengo/grpc-gateway` instead of `github.com/grpc-ecosystem/grpc-gateway`, means fetching it as a dependency is currently broken as `github.com/gengo/grpc-gateway` was moved (and thus no `go.mod` can be fetched).

```
go: sourcegraph.com/sourcegraph/prototools/cmd/protoc-gen-doc imports
        sourcegraph.com/sourcegraph/prototools/tmpl imports
        github.com/gengo/grpc-gateway/protoc-gen-grpc-gateway/descriptor: github.com/gengo/grpc-gateway@v1.14.5: parsing go.mod:
        module declares its path as: github.com/grpc-ecosystem/grpc-gateway
                but was required as: github.com/gengo/grpc-gateway
```

**Comment by keegancsmith** [comment]
@ping-localhost Happy to review and merge a PR fixing this.

**Comment by keegancsmith** [comment]
fixed in 4ba24e49a56785be77b572eb29541d86fec926a7
